src/index.ts

```diff
@@ -122,7 +122,9 @@ export function ShortcutButtonsPlugin(config: ShortcutButtonsFlatpickr.Config) {
         function setButtonsAttributes(button: HTMLButtonElement, attributes?: ShortcutButtonsFlatpickr.Attributes) {
             Object.keys(attributes).filter((attribute) => supportedAttributes.has(attribute)).forEach((key) => {
                 if (key === 'class') {
-                    button.classList.add(attributes[key]);
+                    attributes[key].split(' ').forEach((className) => {
```

Comment:
@hracik this could be simplified to something like the following instead:

```ts
button.classList.add(...attributes[key].split(' '));
```

Other than that this PR looks good 👍